### PR TITLE
Chore/hn 124/multiple llms

### PR DESCRIPTION
**Pull request by TianYao12**
added config stuff for multiple llms

### CODE REVIEW

**Comment by TianYao12** [comment]
@ToJen can u check the PR
As requests u said we will leave Ollama integration for later ticket

File: hive_agent/config/hive_config.toml [hunk 1]
```diff
@@ -3,3 +3,9 @@ model = "gpt-4-turbo-preview"
 
 [environment]
 type = "dev"
+
+
+# claude-3-opus-20240229
+# gpt-4-turbo-preview
+# open-mixtral-8x22b
+
```
Comment (ToJen) [inline hive_agent/config/hive_config.toml:11]: delete unused code

File: hive_agent/agent.py [hunk 2]
```diff
@@ -63,31 +73,28 @@ def _check_optional_dependencies(self):
...
-            Here is your domain-specific instruction:
-            {self.instruction}
-            """
-        )
+        model = config.get("model", "model", "gpt-3.5-turbo")
+        if "gpt" in model:   
+            self.__agent = OpenAILLM(tools, self.instruction).agent
+        elif "claude" in model: 
+            self.__agent = ClaudeLLM(tools, self.instruction).agent 
+        elif "ollama" in model:
```
Comment (ToJen) [inline hive_agent/agent.py:89]: should be `elif "llama"...`

File: hive_agent/server/routes/chat.py [hunk 3]
```diff
@@ -36,12 +36,19 @@ async def chat(request: Request, data: ChatData):
...
+            response = await agent.achat(last_message.content, messages)
 
         async def event_generator():
             async for token in response.async_response_gen():
                 if await request.is_disconnected():
                     break
                 yield token
-
-        return StreamingResponse(event_generator(), media_type="text/plain")
+        if "gpt" in model:
```
Comment (ToJen) [inline hive_agent/server/routes/chat.py:51]: you should check the type of `agent`

**Review by ToJen — CHANGES_REQUESTED** [review]
please see comments

**Comment by TianYao12** [comment]
there was a weird thing with achat where returning chat response resulted in '"chat response"', but for assertion tests I think it's fine as we're just testing for achat vs streaming right, like which function it was in, the actual response message is not that important

**Review by ToJen — APPROVED** [review]

**Pull request merged**